src/routes/test/edit/BlocksEditor.svelte

```diff
@@ -239,15 +237,35 @@
 			});
 	}
 
+	function runTestInCloud(projectId: string) {
```

Comment:
If you can merge this this evening, then I can add it into the new block editor.